src/services/upper-bar-manager/upper-bar-manager.tsx

```diff
@@ -76,6 +76,20 @@ export class UpperBarManager {
     return icons.sort((a, b) => iconsOrder[a.displayName] - iconsOrder[b.displayName]);
   }
 
+  public focusPluginButton(pluginName: string) {
+    let pluginButton;
+    const controls = this.getControls(this.iconsOrder);
+    const pluginElement = controls.find((control) => control.displayName === pluginName)?.componentRef?.current
+      ?.base as HTMLElement;
+    if (pluginElement) {
+      pluginButton = pluginElement.querySelector('button');
+    } else {
+      const rightControls = this.displayedBarComponentRefs.Playback.current?.base as HTMLElement;
+      const buttons = rightControls.getElementsByTagName('button');
```

Comment:
probably better find not a `<button>` tag, but first element with tab-index 0, since we can use f.e. `<div>` with role=button

Reply:
done